libcylinder/src/key.rs

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
I think you are missing a word in your commit message. Should be "These structs will be used throughout the library."

Reply:
Fixed